### PR TITLE
Add Data Studios environment image versions

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/issues/PLAT-432
- https://seqera.atlassian.net/browse/EDU-316

Specify the latest versions of the images that Data Studios ships. This happens out-of-band from releases.

### CODE REVIEW

**Comment by jason-seqera** [comment]
@weronikasosnowskaseqera, do we know if it is `public.cr.seqera.io` or `cr.seqera.io`? Thanks!

**Comment by weronikasosnowskaseqera** [comment]
@jason-seqera It will be `public.cr.seqera.io `, in `cr.seqera.io` we store images for dev and staging and in this case we should list only prod images

**Comment by jason-seqera** [comment]
@justinegeffen @llewellyn-sl 

This is one possible placement for surfacing the Data Studios environment images. The other is in: `platform_versioned_docs/version-24.1/data/data-studios.mdx`.

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
> @justinegeffen @llewellyn-sl
> 
> This is one possible placement for surfacing the Data Studios environment images. The other is in: `platform_versioned_docs/version-24.1/data/data-studios.mdx`.

Let's wait until the IA changes are merged because that'll change a lot of things.

File: platform_versioned_docs/version-24.1/enterprise/data-studios.mdx
Comment (weronikasosnowskaseqera) [inline platform_versioned_docs/version-24.1/enterprise/data-studios.mdx:43]: Just so you know, we also have an old RStudio version 4.0.0. @jimmypoms Do we want to list all of them here?

**Review by weronikasosnowskaseqera — APPROVED** [review]
Links are correct, I would double check with product if we don't want to list in docs old rstudio version as an information that we support both.

File: platform_versioned_docs/version-24.1/enterprise/data-studios.mdx
Comment (jimmypoms) [reply platform_versioned_docs/version-24.1/enterprise/data-studios.mdx:43]: Not sure :thinking: @seqeralabs/scientific-development is there value in having both rstudio 4.0 and rstudio 4.4 as data studio templates?

Another small thing: please use versions `major.minor` for the data studio client throughout the documentation (e.g. `public.cr.seqera.io/platform/data-studio-rstudio:4.4.1-0.7`). It makes our lives easier :)

Comment (pinin4fjords) [reply platform_versioned_docs/version-24.1/enterprise/data-studios.mdx:43]: I think if we want these to be at all reproducible for people it's hard to justify deleting prior images. Though the list could get long over subsequent updates, so I'm not sure of the right path.

Comment (jason-seqera) [reply platform_versioned_docs/version-24.1/enterprise/data-studios.mdx:43]: @pinin4fjords, so for this, we've wanted to start exploring using the release notes for listing new images as they come out; they challenge is right now, we don't have an async release note set yet.

cc @justinegeffen @llewellyn-sl

**Pull request merged**
